devicetypes/smartthings/zigbee-rgbw-bulb.src/zigbee-rgbw-bulb.groovy

```diff
@@ -133,9 +133,10 @@ def refresh() {
 
 def configure() {
     log.debug "Configuring Reporting and Bindings."
-    // Enrolls device to Device-Watch with 3 x Reporting interval 30min
+    // Device-Watch allows 3 check-in misses from device. 300 seconds x 3 = 15min
     sendEvent(name: "checkInterval", value: 1800, displayed: false, data: [protocol: "zigbee"])
```

Comment:
Should this be 
`sendEvent(name: "checkInterval", value: 900, displayed: false, data: [protocol: "zigbee"])` instead?